### DISCUSSION TITLE
question about the list of destination directories.

### DISCUSSION THREAD

**Discussion by ffffffff01** [swar/Swar-Chia-Plot-Manager#124] [Q&A]
If I have a list of destination directories, will the tool judge whether the directory has enough space before writing? And if there's not enough space for the first directory, it will try the next one, or the tool will just select one directory to write by cycle regardless whether the space is enough? Thanks.

**Comment by raid5** [comment]
@swar recently mentioned on Keybase that it currently just rotates through the destination drives without taking remaining space into account. It is a feature that is planned to be worked on at some point though.

**Reply by ffffffff01** [reply]
Thanks a lo!
